cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -65,12 +65,15 @@ mode and use a `cloud-init` disk, the virtual machine will lose its network
 connectivity after being restarted. As a workaround, remove the `HWADDR` line
 in the file `/etc/sysconfig/network-scripts/ifcfg-eth0`.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1708680[*BZ#1708680*])
-+
-[NOTE]
-====
-Avoid this issue by using the recommended `masquerade` binding method for
-the default Pod network.
-====
+
+* Masquerade does not currently work with CNV. Due to this limitation,
+you cannot connect a virtual machine to the default Pod network while in Masquerade mode.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1725848[*BZ#1725848*])
```

Comment:
The bug you cite is about a gui-only problem. Fir the urgent issue which breaks masquerade everywhere, I'm aware only of an upstream issue https://github.com/kubevirt/kubevirt/issues/2400

Reply:
Thanks Dan. I'll swap that link out for the one you supply re this issue and update this PR shortly.

Bob

Reply:
Made change. Not sure if we allow github links in Rel Notes (I have a question out about that). @dankenigsberg let me know what you think about what I have done and I will merge.

Bob